ix-dev/community/tailscale/questions.yaml

```diff
@@ -64,15 +65,21 @@ questions:
           label: Advertise Exit Node
           description: |
             Advertise exit node</br>
-            Same as `--advertise-exit-node` flag.
+            Same as `--advertise-exit-node` flag.</br>
+            Needs enabled IP forwarding on the host via System > Advanced Settings > Sysctls.</br>
+            Please make sure you read and understand the warnings displayed when adding Sysctls</br>
+            See also https://tailscale.com/kb/1019/subnets?tab=linux#enable-ip-forwarding
           schema:
             type: boolean
             default: false
         - variable: advertise_routes
           label: Advertise Routes
           description: |
             Advertise routes</br>
-            Same as `--advertise-routes` flag.
+            Same as `--advertise-routes` flag.</br>
```

Comment:
Unfortunately this whole description blog is missing :-(

![image](https://github.com/user-attachments/assets/bd361cfa-3ca9-4f4f-8ecd-84d271cf50f6)